### PR TITLE
Slip-a-ping: Add basic SLIP-0044 coin type info to base assets

### PR DESCRIPTION
**Pull request by Shadowfiend**
(Originally discussed as part of the still-feature-flagged [RNS work](https://github.com/tallycash/extension/pull/1366#discussion_r883694240).
Note that this PR does NOT update any name resolvers to use `coinType`.)

Network base assets now have a SLIP44 coinType[^1] field, and a very basic
mapping is added from symbol to coin type. Only coin types currently
referenced in network base assets are tracked.

As noted in the coin-types file, although the data could be imported
from an external package, it is deliberately inlined here as plain data
to avoid pulling in external executable code for something that
represents simple data.

[^1]: https://github.com/satoshilabs/slips/blob/master/slip-0044.md

### CODE REVIEW

File: background/constants/networks.ts [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import { EVMNetwork, Network } from "../networks"
 import { BTC, ETH, MATIC } from "./currencies"
 
-// TODO integrate this with /api/networks
```
Comment (mhluongo) [inline background/constants/networks.ts:4]: 🙏

File: background/constants/coin-types.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// This file includes SLIP-0044 data used by the extension. Because this is raw
```
Comment (mhluongo) [inline background/constants/coin-types.ts:1]: I like this approach.

**Review by mhluongo — DISMISSED** [review]
This looks awesome. Concept ACK on mobile, looking for another +1 and test to merge.

**Comment by mhluongo** [comment]
Also looking for green tests, ofc :)

**Comment by Shadowfiend** [comment]
Sorry, pushed a fix for the tests :facepalm:

**Review by greg-nagy — APPROVED** [review]
This PR is georgous

Let's 🚢

**Review by greg-nagy — APPROVED** [review]
and again

**Pull request merged**